### PR TITLE
Add option to save credentials

### PR DESCRIPTION
**Pull request by haggaret**
Store the AWS credentials that are used to access parameter store to the given filename (in the credentials path) with the given profile name

### CODE REVIEW

File: parameter_sync.py [hunk 1]
```diff
@@ -167,4 +187,43 @@ def get_parameters_with_prefix(prefix, next_token=None):
...
+
+        if len(args.save_aws_creds) % 2 != 0:
+            logging.critical('Must provide both filename and profile name for aws creds')
+            sys.exit(1)
+
+        aws_creds_tuples = []
+        filename = None
+        profile = None
+        for i, val in enumerate(args.save_aws_creds):
+            if (i % 2) == 0:
```
Comment (haggaret) [inline parameter_sync.py:207]: The idea here is that if you use the --save-aws-creds option, you need to provide two inputs, the filename to save to and the profile name. The first input is the filename, the second in the profile name

File: parameter_sync.py [hunk 2]
```diff
@@ -163,8 +186,54 @@ def get_parameters_with_prefix(prefix, next_token=None):
...
+
+    if args.save_aws_creds:
+        if args.aws_access_key and args.aws_secret_key:
+            aws_access_key = args.aws_access_key
+            aws_secret_key = args.aws_secret_key
+        else:
+            aws_access_key = os.environ.get('AWS_ACCESS_KEY_ID')
+            aws_secret_key = os.environ.get('AWS_SECRET_ACCESS_KEY')
+
+        if len(args.save_aws_creds) % 3 != 0:
```
Comment (haggaret) [inline parameter_sync.py:252]: The idea here is that if you use the --save-aws-creds option, you need to provide three inputs, the filename to save to, the profile name and the region (or None). The first input is the filename, the second is the profile name, and the third is the region

**Review by sigJoe — APPROVED** [review]
Not sure what prompted this change, but sounds legit

**Comment by haggaret** [comment]
> Not sure what prompted this change, but sounds legit

We were previously syncing the AWS Creds by writing them to the parameter store, but also using the same creds to do the sync from the parameter store - since we already have them, why not just write them to disk instead of pulling them from the parameter store. 🙂

**Review by miketypeguy — APPROVED** [review]

**Pull request merged**